extras_install.sh

```diff
@@ -7,7 +7,7 @@ echo "Platform is ${os_name}, Version: ${os_version}"
 ID=$(cat /etc/os-release | grep -w ID | cut -f2 -d"=")
 
 if [ ${ID} != "mendel" ]; then
-  if [[  $os_name == *"Red Hat"* || $os_name == *"CentOS"*  ]]; then
+  if [[ "$os_name" == *"Red Hat"* ]] || [[ "$os_name" == *"CentOS"* ]]; then
```

Comment:
` if [[ "$os_name" == *"Red Hat"*  ||  "$os_name" == *"CentOS"* ]]; then`  should work fine.

Reply:
not working

Reply:
OK, go ahead with whatever works!